### PR TITLE
Reduce player softcap for wizden servers and panic bunker wizden Levi

### PR DESCRIPTION
**Pull request by nikthechampiongr**
## About the PR

This pr changes the softplayer cap to 70 for wizden servers and enables panic bunker by default on leviathan.

## Why / Balance

Following discussions with the admin team, it was decided that these changes will help alleviate admin load by making admining the servers more manageable which is severely needed. 

Imo 70 players is also a better player experience.
## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

no cl no fun

### CODE REVIEW

**Comment by Cojoke-dot** [comment]
This… kinda sucks but I'm not an admin so I don't know your workload. What I want to bring up is the number of people who sit in lobbies afk without joining for multiple rounds. Currently lizard is full at 80/80 most evenings when I get on. It ends up being about 60 people in shift and ~5 spectators. Dropping that by ten is not going to affect the people who afk in the lobby for multiple shifts in a row. There needs to be some kind of system to auto kick people if they afk for 30 minutes in the lobby so there aren't people just saving spots for hours.

**Comment by Plykiya** [comment]
Non-admin opinion but surely reducing the playercap by 10 doesn't reduce the amount of problematic ahelps that occur by any significant degree?

There are servers that get raided unanswered at around 20-30 pop, and expecting admins to be able to respond to that kind of thing for 6/7 servers at all times feels like the bigger issue. Reducing the amount of servers that Wizden has to a point where the amount of available admins to answer ahelps is at a more consistent level feels like the ideal end goal?

**Comment by nikthechampiongr** [comment]
> Non-admin opinion but surely reducing the playercap by 10 doesn't reduce the amount of ahelps, and doesn't increase the availability of admins?

The idea behind it is that 70 players compared to 80 players does reduce ahelps for normal issues as it reduces chaos, and generally leads to more behaved rounds. It does not have an impact on raider issues as you said.

> Reducing the amount of servers that Wizden has to a point where the amount of available admins to answer ahelps is at a more consistent level feels like the ideal end goal?
This was also discussed but ultimately decided again. We might take this measure in the future.

**Comment by nikthechampiongr** [comment]
> This… kinda sucks but I'm not an admin so I don't know your workload. What I want to bring up is the number of people who sit in lobbies afk without joining for multiple rounds. Currently lizard is full at 80/80 most evenings when I get on. It ends up being about 60 people in shift and ~5 spectators. Dropping that by ten is not going to affect the people who afk in the lobby for multiple shifts in a row. There needs to be some kind of system to auto kick people if they afk for 30 minutes in the lobby so there aren't people just saving spots for hours.

If you wish to code something to deal with afk players in the lobby then feel free to do so, it would most likely be fine. There is already a system to prevent afk time farming for role timers.

**Comment by Golinth** [comment]
So, I'm not sure what the accomplishes to be honest. If the issue is too many players, this doesnt do anything since the players that cant join due to the 70 player cap will just join vulture or miros instead. The capacity for players is still the same, they will just migrate to other servers. I do not see how admin workload will meaningfully decrease from this change. I'd rather see servers be removed, minimum admin hours be implemented, or more admins be brought onto the team before this. 

(and for my super hot take, remove salamander entirely. The split load of admins needing to specialize in MRP and LRP rulesets, game design split, and rules needing to be split between both versions, make the entire wizden experience worse. Almost every single popular non-wizden is MRP, so the players have places to go.)

**Review by chromiumboy — APPROVED** [review]

**Comment by ARMOKS** [comment]
Just hire more admins lmao

**Comment by ARMOKS** [comment]
> Just hire more admins lmao

the 70 people change is a bandaid on a what is a long term issue that needs to be resolved, AKA admin burnout and admin inactivity. Solving those should be the first step.

**Comment by mirrorcult** [comment]
thanks for the info, the admin team probably had no idea about that

**Comment by ARMOKS** [comment]
Ignoring the unproductive comment, a few solutions that would genuinely work to fix the systematic problems could be as follows;

1) Implementation of easier to enforce rules
 - Currently there exist multiple rules that are difficult to both understand and enforce, as well as causing more issues than they resolve. TG stations rules are a great implementation on this and is worth giving a look.

2) Require administrators to admin for the amount of time they claim they will in their application. 
 - This will likely be a controversial one due to the "we are volunteers" argument, but as a counterpoint, I regularly volunteer at a food bank in my area, if I started being extremely inconsistent in my volunteering it would screw over the rest of my teammates and damage the center as a whole. For this exact reason, the center does not accept inconsistent help. There is no moral issue with that. Additionally, inconsistent assistance puts more load on the admins actively working on servers, several come to mind, which is unfair and doubly so burning out for them,

3) Hire more admins
 - There is currently a lack of admins, this is well known both within and outside the administration team. There are PLENTY of people who want to help, but due to either admin applications being closed or lack of advertisement, are not being recruited. No one would mind a @everyone in the discord asking for help. Additionally, unless I am misinformed, the mentoring of trialmins is currently done on a one-by-one basis, rather than in batches; leading to an even slower admin production rate.

4) I could be mistaken, but I do not believe the administrator team coordinates times for people to be online. Doing so would further alleviate the stress of constantly reacting to raiders rather than preemptively dealing with such issues.

Furthermore, the presented solution will not help and I guarantee that it will not change anything, here are some reasons;

1) The slots will most likely be filled with problematic roles, AKA tiders. On 80 pop command and security are already often missing, this will only worsen with a lower pop and generate more issues.

2) A -10 pop will not prevent problem players from joining and causing problems, they will still be generating ahelps.

3) Finally, reducing the pop will only make said removed players move to the currently unpopulated servers, miros, and vulture. On top of not changing the amount of players that need administration; This could potentially have the opposite of the desired effect, by making those servers have a desirable population for more players to join. Several groups I am a part of have already discussed moving as a whole to said servers, after this change, so it is not an unlikely option.

I am open to criticism if anyone has an issue with the reasons I have brought up. I can practically guarantee however this will not have the desired effect.

**Comment by ARMOKS** [comment]
> Ignoring the unproductive comment, a few solutions that would genuinely work to fix the systematic problems could be as follows;
> 
> 1. Implementation of easier to enforce rules
> 
> * Currently there exist multiple rules that are difficult to both understand and enforce, as well as causing more issues than they resolve. TG stations rules are a great implementation on this and is worth giving a look.
> 
> 2. Require administrators to admin for the amount of time they claim they will in their application.
> 
> * This will likely be a controversial one due to the "we are volunteers" argument, but as a counterpoint, I regularly volunteer at a food bank in my area, if I started being extremely inconsistent in my volunteering it would screw over the rest of my teammates and damage the center as a whole. For this exact reason, the center does not accept inconsistent help. There is no moral issue with that. Additionally, inconsistent assistance puts more load on the admins actively working on servers, several come to mind, which is unfair and doubly so burning out for them,
> 
> 3. Hire more admins
> 
> * There is currently a lack of admins, this is well known both within and outside the administration team. There are PLENTY of people who want to help, but due to either admin applications being closed or lack of advertisement, are not being recruited. No one would mind a @everyone in the discord asking for help. Additionally, unless I am misinformed, the mentoring of trialmins is currently done on a one-by-one basis, rather than in batches; leading to an even slower admin production rate.
> 
> 4. I could be mistaken, but I do not believe the administrator team coordinates times for people to be online. Doing so would further alleviate the stress of constantly reacting to raiders rather than preemptively dealing with such issues.
> 
> Furthermore, the presented solution will not help and I guarantee that it will not change anything, here are some reasons;
> 
> 1. The slots will most likely be filled with problematic roles, AKA tiders. On 80 pop command and security are already often missing, this will only worsen with a lower pop and generate more issues.
> 2. A -10 pop will not prevent problem players from joining and causing problems, they will still be generating ahelps.
> 3. Finally, reducing the pop will only make said removed players move to the currently unpopulated servers, miros, and vulture. On top of not changing the amount of players that need administration; This could potentially have the opposite of the desired effect, by making those servers have a desirable population for more players to join. Several groups I am a part of have already discussed moving as a whole to said servers, after this change, so it is not an unlikely option.
> 
> I am open to criticism if anyone has an issue with the reasons I have brought up. I can practically guarantee however this will not have the desired effect.

File: Resources/ConfigPresets/WizardsDen/wizardsDen.toml [hunk 1]
```diff
@@ -4,12 +4,12 @@
...
 desc             = "Official English Space Station 14 servers. Vanilla, roleplay ruleset."
 lobbyenabled     = true
-soft_max_players = 80
+soft_max_players = 70
 panic_bunker.enabled = true
 panic_bunker.disable_with_admins = true
 panic_bunker.enable_without_admins = true
 panic_bunker.show_reason = true
-panic_bunker.custom_reason = "You have not played on a Wizard's Den server long enough to connect to this server. Please play on Wizard's Den Lizard, Leviathan, or Farm Grass Hopper until you have more playtime."
+panic_bunker.custom_reason = "You have not played on a Wizard's Den server long enough to connect to this server. Please play on Wizard's Den Lizard, or Farm Grass Hopper until you have more playtime."
```
Comment (slarticodefast) [inline Resources/ConfigPresets/WizardsDen/wizardsDen.toml:12]:
```suggestion
panic_bunker.custom_reason = "You have not played on a Wizard's Den server long enough to connect to this server. Please play on Wizard's Den Lizard or Farm Grass Hopper until you have more playtime."
```
No Oxford comma in a list with only two entries 🙂

**Pull request merged**